README.md

```diff
@@ -362,8 +362,9 @@ Commands:
 ```
 
 You can create a [GCP Notebook](https://cloud.google.com/vertex-ai/docs/workbench/notebook-solution) controlled
-resource with `terra resource create gcp-notebook`. These `stop`, `start` commands are provided for convenience.
-You can also stop and start the notebook using the `gcloud notebooks instances start/stop` commands.
+resource with `terra resource create gcp-notebook --name=resourcename [--workspace=workspaceid]`. These `stop`, `start`
```

Comment:
nit: Recommend wrapping the parameter values in `<>` to be consistent with CLI help format. `--name=<resourcename> [--workspace=<id>]`.

Reply:
Done